controllers/virtualmachinegrouppublishrequest/virtualmachinegrouppublishrequest_controller.go

```diff
@@ -56,10 +69,12 @@ func AddToManager(ctx *pkgctx.ControllerManagerContext, mgr manager.Manager) err
 				controlledType,
 				mgr.GetScheme()),
 		}).
+		Watches(&vmopv1.VirtualMachinePublishRequest{},
+			handler.EnqueueRequestForOwner(mgr.GetScheme(), mgr.GetRESTMapper(), &vmopv1.VirtualMachineGroupPublishRequest{})).
```

Comment:
I think this should include `handler.OnlyControllerOwner()` since we're using controlling owner refs on these